src/light-godwoken/constants/tokens.ts

```diff
@@ -145,3 +145,18 @@ export const getTokenList = () => {
     v1: TOKEN_LIST_V1,
   };
 };
+
+export function toHumanReadable(uan: string | undefined): string {
+  if (!uan) {
+    return "";
+  }
+  const dotRegex = new RegExp(/\./g);
+  const splitRegex = new RegExp(/\|/g);
+  const fbRegex = new RegExp(/fb/g);
+  const gbRegex = new RegExp(/gb/g);
+  return uan
+    .replace(dotRegex, " on ")
+    .replace(splitRegex, " from ")
+    .replace(fbRegex, "Force Bridge")
+    .replace(gbRegex, "Godwoken Bridge");
+}
```

Comment:
```suggestion
// TODO transform UAN to a structure data to avoid translate "fb" token to Force Bridge 
export function toHumanReadable(uan: string | undefined): string {
  if (!uan) {
    return "";
  }
  const dotRegex = new RegExp(/\./g);
  const splitRegex = new RegExp(/\|/g);
  const fbRegex = new RegExp(/fb/g);
  const gbRegex = new RegExp(/gb/g);
  return uan
    .replace(dotRegex, " on ")
    .replace(splitRegex, " from ")
    .replace(fbRegex, "Force Bridge")
    .replace(gbRegex, "Godwoken Bridge");
}
```